hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/records/RecordsHistorian.java

```diff
@@ -136,6 +136,21 @@ void trackFollowingChildRecord(
             ExpirableTxnRecord.Builder recordSoFar,
             List<TransactionSidecarRecord.Builder> sidecars);
 
+    /**
+     * Adds the given in-progress child record to the active transaction, where its synthetic
+     * consensus timestamp will come <i>after</i> that of the parent user transaction.
```

Comment:
Does this mean _immediately_ after, i.e., first?

Reply:
None of the mono-service changes in this PR would be worth putting into production in any case; they are immaterial changes needed for diff testing on `release/0.48`.